packages/backend/index.js

```diff
@@ -390,6 +390,83 @@ app.get("/challenges", withRole("admin"), async (request, response) => {
   }
 });
 
+app.post("/api/frame", (req, res) => {
+  const baseUrl = "https://speedrunethereum.com";
+  const BG_SERVER = process.env.BG_BACKEND;
```

Comment:
I got this `.env` variable from another backend file. Please let me know if this is correct and will result in the correct backend url in `production`

Reply:
Just hardcoded the real backend URL (dedicated one for SRE)